### ISSUE TITLE
Azurophil granule - has_part relationships too liberal?

### ISSUE THREAD

**Issue by dosumis** [obophenotype/cell-ontology#2309]
According to GO, azurophil granules are only found in neutrophils, but CL has them as part of many cell types that are not classified as neutrophil.  Who is wrong?

<img width="722" alt="image" src="https://github.com/obophenotype/cell-ontology/assets/112839/f8dc64ed-cf07-4ee9-be4e-57cc318b7540">

CC @addiehl

**Comment by addiehl** [comment]
GO is too limited in their definition of [azurophil granule](https://amigo.geneontology.org/amigo/term/GO:0042582).  Per wikipedia: "An azurophilic granule is a cellular object readily [stainable](https://en.wikipedia.org/wiki/Staining) with a [Romanowsky stain](https://en.wikipedia.org/wiki/Romanowsky_stain). In [white blood cells](https://en.wikipedia.org/wiki/White_blood_cell) and [hyperchromatin](https://en.wikipedia.org/wiki/Chromatin), staining imparts a [burgundy](https://en.wikipedia.org/wiki/Burgundy_(color)) or merlot coloration. [Neutrophils](https://en.wikipedia.org/wiki/Neutrophil) in particular are known for containing azurophils loaded with a wide variety of anti-microbial [defensins](https://en.wikipedia.org/wiki/Defensin) that fuse with [phagocytic](https://en.wikipedia.org/wiki/Phagocytic) [vacuoles](https://en.wikipedia.org/wiki/Vacuole).

Per [Quora](https://www.quora.com/What-are-the-specific-granules-and-the-Azurophilic-granules-What-do-they-do-and-whats-the-difference-between-them) (not necessarily a perfect source): "Azurophilic (nonspecific) granules are lysosomes found in the cytoplasm of all five kinds of white blood cells. They’re named for their property of staining with blue or violet dyes (azur = blue; philic = loving)."

[PMID:5914694](https://pubmed.ncbi.nlm.nih.gov/5914694/) describes in great detail the staining of developmental (and mature stages of granulocytes and describes azurophil granules in progranulocytes, myelocytes, metamyelocytes, band cells, and mature PMN (neutrophils, basophils, and eosinophils).

Separately, NK cells [are described](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/azurophilic-granule) as having azurophilic granules, and some T cells as well ([PMID:28717070](https://pubmed.ncbi.nlm.nih.gov/28717070/) talks about normal NK and T cells have such granules, which are found also in many leukemias).

Basically, GO should redefine their definition of azurophil granule to be more general.  Perhaps, something derived from Wikipedia's description:
"An azurophilic granule is a primary lysosome readily stainable with a Romanowsky stain."
They would probably hate that.

I think CL is fine.

**Comment by addiehl** [comment]
GO has now implemented my suggestion to broaden their definition of azurophil granule, via https://github.com/geneontology/go-ontology/pull/27271#event-12110297865
I will close this issue now.

**Comment by dosumis** [comment]
Thanks,  good that incorrect GO axiom removed.
